Content.Shared/Damage/DamageSystem.cs

```diff
@@ -9,59 +9,6 @@ namespace Content.Shared.Damage
     [UsedImplicitly]
     public class DamageSystem : EntitySystem
     {
-        public static ImmutableDictionary<DamageClass, ImmutableList<DamageType>> ClassToType { get; } = DefaultClassToType();
 
-        public static ImmutableDictionary<DamageType, DamageClass> TypeToClass { get; } = DefaultTypeToClass();
-
-        private static ImmutableDictionary<DamageClass, ImmutableList<DamageType>> DefaultClassToType()
-        {
-            return new Dictionary<DamageClass, ImmutableList<DamageType>>
-            {
-                [DamageClass.Brute] = new List<DamageType>
-                {
-                    DamageType.Blunt,
-                    DamageType.Slash,
-                    DamageType.Piercing
-                }.ToImmutableList(),
-                [DamageClass.Burn] = new List<DamageType>
-                {
-                    DamageType.Heat,
-                    DamageType.Shock,
-                    DamageType.Cold
-                }.ToImmutableList(),
-                [DamageClass.Toxin] = new List<DamageType>
-                {
-                    DamageType.Poison,
-                    DamageType.Radiation
-                }.ToImmutableList(),
-                [DamageClass.Airloss] = new List<DamageType>
-                {
-                    DamageType.Asphyxiation,
-                    DamageType.Bloodloss
-                }.ToImmutableList(),
-                [DamageClass.Genetic] = new List<DamageType>
-                {
-                    DamageType.Cellular
-                }.ToImmutableList()
-            }.ToImmutableDictionary();
-        }
-
-        private static ImmutableDictionary<DamageType, DamageClass> DefaultTypeToClass()
-        {
-            return new Dictionary<DamageType, DamageClass>
-            {
-                {DamageType.Blunt, DamageClass.Brute},
-                {DamageType.Slash, DamageClass.Brute},
-                {DamageType.Piercing, DamageClass.Brute},
-                {DamageType.Heat, DamageClass.Burn},
-                {DamageType.Shock, DamageClass.Burn},
-                {DamageType.Cold, DamageClass.Burn},
-                {DamageType.Poison, DamageClass.Toxin},
-                {DamageType.Radiation, DamageClass.Toxin},
-                {DamageType.Asphyxiation, DamageClass.Airloss},
-                {DamageType.Bloodloss, DamageClass.Airloss},
-                {DamageType.Cellular, DamageClass.Genetic}
-            }.ToImmutableDictionary();
-        }
```

Comment:
![718132303183872010 (1)](https://user-images.githubusercontent.com/10968691/125179354-4ae63900-e1ee-11eb-8735-16ed526da714.png)

I'm so sorry.

Reply:
Why are you sorry Dr. PHD Smugleaf?

Reply:
give me two months eve sub and ill give you a four hour crash course on why smug is sorry 

comes with a free tutorial of IEnumerable examples that crash my pc